File: GuardianMonsters/core/src/de/limbusdev/guardianmonsters/fwmengine/world/ecs/components/GuardoSphereComponent.java
package de.limbusdev.guardianmonsters.fwmengine.world.ecs.components;

import com.badlogic.ashley.core.Component;

import de.limbusdev.guardianmonsters.guardians.monsters.GuardoSphere;

public class GuardoSphereComponent implements Component
{
    public GuardoSphere guardoSphere;

    public GuardoSphereComponent()
    {
        guardoSphere = new GuardoSphere();
    }
}

File: GuardianMonsters/core/src/de/limbusdev/guardianmonsters/fwmengine/world/ecs/components/GuardoSphereComponent.kt
package de.limbusdev.guardianmonsters.fwmengine.world.ecs.components

import de.limbusdev.guardianmonsters.fwmengine.world.ecs.LimbusBehaviour

import de.limbusdev.guardianmonsters.guardians.monsters.GuardoSphere

/**
 * @author Georg Eckert 2019-07-03
 */
class GuardoSphereComponent : LimbusBehaviour()
{
    companion object
    {
        const val className ="GuardoSphereComponent"
    }

    var guardoSphere = GuardoSphere()
}
